earthaccess/results.py

```diff
@@ -106,17 +107,16 @@ def get_umm(self, umm_field: str) -> Union[str, Dict[str, Any]]:
         return ""
 
     def concept_id(self) -> str:
-        """
-        Returns:
-            A collection's `concept_id`.
-                This id is the most relevant search field on granule queries.
+        """Placeholder.
+
+        Returns: A collection's `concept_id`.This id is the most relevant search field on granule queries.
         """
         return self["meta"]["concept-id"]
 
     def data_type(self) -> str:
-        """
-        Returns:
-            The collection data type, i.e. HDF5, CSV etc., if available.
+        """Placeholder.
+
+        Returns: The collection data type, i.e. HDF5, CSV etc., if available.
```

Comment:
This change is actually breaking some documentation:

https://earthaccess.readthedocs.io/en/latest/user-reference/granules/granules/#earthaccess.results.DataGranule.dataviz_links

https://earthaccess--580.org.readthedocs.build/en/580/user-reference/granules/granules/#earthaccess.results.DataGranule.dataviz_links

The return description is no longer being parsed correctly when this change is applied.